docs/vision-usage.rst

```diff
@@ -0,0 +1,190 @@
+Using the Vision API
+====================
+
+Authentication and Configuration
+--------------------------------
+
+- For an overview of authentication in ``gcloud-python``,
+  see :doc:`gcloud-auth`.
+
+- In addition to any authentication configuration, you should also set the
+  :envvar:`GCLOUD_PROJECT` environment variable for the project you'd like
+  to interact with. If you are Google App Engine or Google Compute Engine
+  this will be detected automatically.
+
+- After configuring your environment, create a
+  :class:`Client <gcloud.vision.client.Client>`
+
+  .. doctest::
+
+     >>> from gcloud import vision
+     >>> client = vision.Client()
+
+  or pass in ``credentials`` and ``project`` explicitly
+
+  .. doctest::
+
+     >>> from gcloud import vision
+     >>> client = vision.Client(project='my-project', credentials=creds)
+
+Annotating an Image
+-------------------
+
+Annotate a single image
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. doctest::
+
+    >>> from gcloud import vision
+    >>> client = vision.Client(project="my-project-name")
+    >>> with open('/tmp/car.jpg', 'r') as f:
+    ...     client.annotate(f.read(), vision.FeatureTypes.LABEL_DETECTION, 3)
```

Comment:
Hm - this feels a bit explicit and tightly coupled to the API. What is the last parameter?

I'd expect this to be something like....

```python
with open('/path/to/image.jpg', 'r') as image:
  client.annotate(image, labels=True, max_labels=3)
```

/cc @quartzmo